docs/releases/5.2.md

```diff
@@ -1,6 +1,6 @@
 # Wagtail 5.2 release notes - IN DEVELOPMENT
```

Comment:
This should also be updated to include (LTS), to match how we've handled other releases (and ensure it shows in the side menu):
```suggestion
# Wagtail 5.2 (LTS) release notes - IN DEVELOPMENT
```

Reply:
Okay. Should  I remove "- IN DEVELOPMENT" here or just add LTS alone?

Reply:
No, leave IN DEVELOPMENT in place - we'll remove that when the release happens.